packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -16,7 +16,11 @@ const packageTag = `@${packageName} `;
 export function moduleFor(description, TestClass, ...generators) {
   let context;
 
-  QUnit.module(`[${packageName}] ${description}`, {
+  let modulePackagePrefixMatch = description.match(/^@(\w*)/); //eg '@glimmer' or '@htmlbars'
+  let modulePackagePrefix = modulePackagePrefixMatch ? modulePackagePrefixMatch[1] : '';
+  let descriptionWithoutPackagePrefix = description.replace(/^@\w* /, '');
```

Comment:
this adds support for module prefixes such as `@glimmer` and `@htmlbars` and is needed as the comprehensive shared tests have highlighted some issues with htmlbars